modules/auxiliary/admin/http/tomcat_utf8_traversal.rb

```diff
@@ -64,7 +65,7 @@ def find_files(files)
       res = send_request_raw(
         {
           'method'  => 'GET',
-          'uri'     => try + files,
+          'uri'     => normalize_uri(datastore['TARGETURI']) + try + files,
```

Comment:
`normalize_uri` should usually encompass all parts of the path. Although I'm not sure if it's relevant in this instance, due to the traversal sequence.

```suggestion
          'uri'     => normalize_uri(datastore['TARGETURI'], try, files),
```